pyanaconda/core/startup/dbus_launcher.py

```diff
@@ -92,7 +92,8 @@ def _write_temporary_config(self):
         if not os.path.exists(dirname):
             os.makedirs(dirname)
 
-        log.info("Writing a temporary configuration loaded from: %s", conf.get_sources())
+        log.info("Configuration loaded from: %s", conf.get_sources())
+        log.info("Writing complete configuration to: %s", ANACONDA_CONFIG_TMP)
```

Comment:
Could you change it to `Writing the runtime configuration to:`?

Reply:
Good idea. Fixed!